### PR TITLE
remove codelines for duplication detection

### PR DESCRIPTION
**Pull request by yoouung**
- changes in `apiRuns.sh` and `README.md` are for deletion
- changes in `txt_proc.py` are for formatting

- modify .gitignore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,21 +21,28 @@
      - `(.venv) % pip install google-api-python-client`
    - Personal Data Detection(Regex)
      - `sample pip install`
-   - Duplication Detection(Konlpy)
-     - `sample pip install`
+   - Duplication Detection
+     - copyKiller (outerText)
+       - `pip install selenium`
+   - Duplication Detection
```
Comment (doorooful) [inline README.md:27]: please remove duplicated lines.

File: README.md [hunk 2]
```diff
@@ -21,21 +21,28 @@
...
+     - copyKiller (outerText)
+       - `pip install selenium`
    - modules/csv_proc
      - `pip install pandas`
 
+<br/>
```
Comment (doorooful) [inline README.md:33]: Could you please remove unnecessary lines?

File: README.md [hunk 3]
```diff
@@ -21,21 +21,28 @@
...
 **API key names**
 
 For the reference, please name environment variable like followings:
 
 - Slang API: SLANG_API_KEY
-- Duplication API: DUP_API_ID, DUP_API_PW
-
-</br>
+- Duplication API: DUC_API_ID, DUC_API_PW
+- Duplication API: DUC_API_ID, DUC_API_PW
```
Comment (doorooful) [inline README.md:45]: Same here.

File: apiRuns.sh [hunk 4]
```diff
@@ -30,18 +30,19 @@ function personalDataDetection {
 }
 
 function duplicationDetection {
-    pddRet=`python duplication/duplication.py --input "${inputText}"`
+    dupRet=`python duplication/duplication.py --input "${inputText}"`
     echo "----Duplication DETECTION----"
-    echo "duplication detection result: ${pddRet}"
+    echo "dupRet is ${dupRet}"
```
Comment (doorooful) [inline apiRuns.sh:35]: Please remove line for debugging purpose

**Review by doorooful — CHANGES_REQUESTED** [review]
Please refer to the comments! ◡̈

**Review by doorooful — CHANGES_REQUESTED** [review]
Please refer to the comments! ◡̈

File: apiRuns.sh (hunk 4 above)
Comment (yoouung) [reply apiRuns.sh]: Got it!

**Review by doorooful — APPROVED** [review]
LGTM!

**Pull request merged**